R/landing_popup_module.R

```diff
@@ -17,11 +17,11 @@
 #' app1 <- init(
 #'   data = teal_data(iris = iris),
 #'   modules = modules(
-#'     example_module()
-#'   ),
-#'   landing_popup = landing_popup_module(
-#'     content = "A place for the welcome message or a disclaimer statement.",
-#'     buttons = modalButton("Proceed")
+#'     example_module(),
```

Comment:
please add deprecated label in the roxy description. Redirect to use `add_landing_modal`. I'd also remove  examples or replace them with `add_landing_modal`.